clang/docs/ReleaseNotes.rst

```diff
@@ -96,6 +96,10 @@ C++20 Feature Support
   behavior can use the flag '-Xclang -fno-skip-odr-check-in-gmf'.
   (`#79240 <https://github.com/llvm/llvm-project/issues/79240>`_).
 
+- Initial support for class template argument deduciton (CTAD) for type alias
+  templates.
+  (`#54051 https://github.com/llvm/llvm-project/issues/54051`_).
```

Comment:
```suggestion
  (#GH54051).
```

Reply:
This doesn't follow the pattern used in this file, we use `#<number> https://github.com/llvm/llvm-project/issues/<nummber>`.

Reply:
That changes recently (either way, you were missing <>)

Reply:
I see, I were not aware of it. Switched to the `GH` form.